### DISCUSSION TITLE
Help ScriptableObject

### DISCUSSION THREAD

**Discussion by DarkHardWorker** [community/community#61088] [Programming Help]
### Select Topic Area

General

### Body

need help puzzling (hi i'm a newbie). I made Weapons and Skins using ScriptableObject, figured out how to select by button, please help me, how to make the button save the selected prefabs from scriptableObject and I can load this character into another scene, most likely calling the object id, etc. In general, please help, try the code, how it's all implemented, thanks

**Comment by dev-fredericfox** [comment]
Wrong forum, you might want to try something Unity realted.
